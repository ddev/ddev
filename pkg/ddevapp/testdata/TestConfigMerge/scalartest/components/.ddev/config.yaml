
database:
  type: mariadb
  version: 10.4
php_version: 8.0
performance_mode: "none"
webserver_type: apache-fpm
