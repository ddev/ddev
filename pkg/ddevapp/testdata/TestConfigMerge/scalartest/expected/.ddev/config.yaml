
database:
  type: mariadb
  version: 10.5
performance_mode: "mutagen"
webserver_type: apache-fpm

php_version: 8.1
